ssl/statem/extensions.c

```diff
@@ -357,11 +357,12 @@ static int extension_is_relevant(SSL *s, unsigned int extctx,
  * found, or an internal error occurred. We only check duplicates for
  * extensions that we know about. We ignore others.
  */
-int tls_collect_extensions(SSL *s, PACKET *packet, unsigned int context,
+size_t tls_collect_extensions(SSL *s, PACKET *packet, unsigned int context,
```

Comment:
If this is a number of extension, not sure it needs to be a size_t; the original int is likely sufficient, given the maximum size of a ClientHello.